ACKReactiveExtensions/Realm/RealmExtensions.swift

```diff
@@ -47,7 +47,7 @@ public extension Reactive where Base: RealmCollection {
             }
 
             func observe() -> NotificationToken? {
-                return self.base.observe(on: nil) { changes in
+                return self.base.observe(on: DispatchQueue(label: "RealmCollectionObserve")) { changes in
```

Comment:
❓ I am not sure we want to create new queue every time this function is called, on the other hand I don't see any reason why not. Just point for discussion, I don't think it is an issue.

Reply:
That's a great question. 👍  I don't think that it's gonna be an issue but I would try it anyway and see the result.

Reply:
Good point, I have changed the implementation to create only a single queue dedicated for `RealmCollection` observing.